sklearn/metrics/tests/test_common.py

```diff
@@ -389,27 +403,29 @@ def test_symmetry():
     y_pred = random_state.randint(0, 2, size=(20, ))
 
     # We shouldn't forget any metrics
-    assert_equal(set(SYMMETRIC_METRICS).union(
+    assert_array_equal(set(SYMMETRIC_METRICS).union(
         NOT_SYMMETRIC_METRICS, THRESHOLDED_METRICS,
         METRIC_UNDEFINED_BINARY_MULTICLASS),
         set(ALL_METRICS))
 
-    assert_equal(
+    assert_array_equal(
```

Comment:
I don't understand this one

Reply:
Good catch. This is an artifact of me horse-blindly removing all occurrences of
`assert_equal` where technically possible. In this situation, it is more clear to keep to original `assert_equal` to avoid confusion.

Will undo this change.